### PR TITLE
Fix qml warnings

### PR DESCRIPTION
**Pull request by FlexW**
For the `AutoSizingMenu` I see no other way to make a Menu that adjusts to the biggest MenuItem. I think the cost of that menu is okay given that not many elements are in that menu.

### CODE REVIEW

File: src/gui/tray/Window.qml [hunk 1]
```diff
@@ -531,24 +529,9 @@ Window {
                                 text: appName
                                 font.pixelSize: Style.topLinePixelSize
                                 icon.source: appIconUrl
-                                width: contentItem.implicitWidth + leftPadding + rightPadding
                                 onTriggered: UserAppsModel.openAppUrl(appUrl)
                                 hoverEnabled: true
 
-                                background: Item {
```
[the remark this answers is not in the document]
Comment (FlexW) [reply src/gui/tray/Window.qml:538]: That is not needed. Check it yourself :) I think that Item was only added to have a hover effect on mouse movement. But the hover effect is there without the Item.

[the remark this answers is not in the document]
Comment (mgallien) [reply src/gui/tray/Window.qml:538]: currently the hover effect is only working when you over the application name and not the whole menu item
does this fix it ?

[the remark this answers is not in the document]
Comment (mgallien) [reply src/gui/tray/Window.qml:538]: cool it fixes that bug

**Comment by mgallien** [comment]
please fix history and we can merge it

**Review by mgallien — APPROVED** [review]

**Review by allexzander — APPROVED** [review]

**Pull request merged**